wiki/Disambiguation/Easy/fr.md

```diff
@@ -0,0 +1,11 @@
+# Easy
+
+**Easy** peut également se référer à :
+
+-   Les [modificateur de jeu](/wiki/Game_Modifiers/#easy).
+-   Le nom de version d'un beatmap.
+-   La difficulté d'une beatmap :
+    -   ![](/wiki/shared/diff/easy-s.png) [Easy](/wiki/Difficulties/Easy), pour osu!standard.
+    -   ![](/wiki/shared/diff/easy-t.png) [Kantan](/wiki/Difficulties/Kantan), pour osu!taiko.
+    -   ![](/wiki/shared/diff/easy-c.png) Cup, pour osu!catch.
```

Comment:
@cl8n Don't we have wiki pages for Cup and EZ?

Reply:
doesn't look like it